### DISCUSSION TITLE
Share settings between Seq instances

### DISCUSSION THREAD

**Discussion by Tthongsy** [datalust/seq-tickets#1288] [Help] [answered]
Hi, 
Is there a way to share settings between Seq instances? 

Thanks,
Tommy T

**Comment by nblumhardt** [comment, marked as answer]
Hi @Tthongsy - thanks for getting in touch.

There's no general out-of-the-box automated way, but some level of synchronization could be achieved by writing code against the _Seq.Api_ C# client (`SeqConnection.Settings` can manipulate many things).

Depending on how much you want to keep two servers in sync, backup + restore could be viable, too, and I've heard of it being used for this quite often.

If there's a specific thing you want to keep in sync, let me know, there could be other options I'm not thinking of.

Best regards,
Nick

**Reply by Tthongsy** [reply]
Hey,
I'm unsure of what other options there may be, but thank you for the suggestions. I'll keep these in mind.

Thanks,
Tommy T
